satisfactory-accounting/src/database.rs

```diff
@@ -44,8 +47,7 @@ impl DatabaseVersion {
     /// Identifies which database versions are considered deprecated.
     pub fn is_deprecated(self) -> bool {
```

Comment:
Deprecated here means that it won't show up in the menu for selection. It's intended to be used for older sub-versions. Basically, only the latest sub-version of each major update should show in the menu by default, and all the other sub-versions should be hidden.

So I still want U5 final and U6 beta to be not-deprecated here since I suspect some people will still be using those for a bit.

(Can you add this information to the doc comment as well for anyone changing this in the future?).